src/pages/de/install/manual.md

```diff
@@ -0,0 +1,143 @@
+
+```bash
+npm init --yes
+```
+
+
+## 2. Installiere Astro
+
+Füge zuerst Astro zu den erforderlichen Paketen deines Projekts hinzu:
+
+```bash
+npm install astro
+```
+
+Ersetze dann jegliche Platzhalter im Abschnitt `scripts` der Datei `package.json` mit folgenden Astro-Skripts:
+
+```diff
+  "scripts": \{
+-    "test": "echo \"Error: no test specified\" && exit 1"
++    "dev": "astro dev",
++    "start": "astro dev",
++    "build": "astro build",
++    "preview": "astro preview"
+  },
+```
+
+Du kannst diese Skripts später verwenden, um Astro zu starten und seine verschiedenen Befehle auszuführen.
+
+
+## 3. Erstelle deine erste Seite
+
+Verwende deinen Texteditor, um im Projektverzeichnis eine neue Datei mit dem Pfad `src/pages/index.astro` zu erstellen. Sie wird die erste Astro-Seite deines Projekts.
+
+Kopiere im Rahmen dieser Anleitung den folgenden Code-Schnipsel (einschließlich der Bindestriche `---`) und füge ihn in deine neue Datei ein:
+
+```astro
+---
+// Willkommen bei Astro! Alles innerhalb dieses mit drei Bindestrichen
+// vom Rest getrennten Abschnitts nennt man "Komponenten-Frontmatter".
+// Der Code hier wird nie im Browser ausgeführt.
+console.log('This runs in your terminal, not the browser!');
+---
+<!-- Nun folgt deine "Komponenten-Vorlage". Sie besteht aus HTML,
+     ergänzt durch einen Hauch an Magie, um dir beim Erstellen
+     großartiger Vorlagen zu helfen. -->
+<html>
+  <body>
+    <h1>Hallo Welt!</h1>
+  </body>
+</html>
+<style>
+  h1 {
+    color: orange;
+  }
+</style>
+```
+
+
+## 4. Erstelle deine erste statische Ressource
+
+Du solltest auch ein `public/`-Verzeichnis erstellen, um deine statischen Ressourcen zu speichern. Astro kopiert diese Ressourcen beim Buildvorgang immer unverändert in das Ausgabeverzeichnis, sodass du sie sicher aus deinen Komponentenvorlagen heraus referenzieren kannst.
+
+Erstelle in deinem Texteditor eine neue Datei mit dem Pfad `public/robots.txt`. Die Datei `robots.txt` wird von den meisten Websites verwendet, um Suchmaschinen wie Google mitzuteilen, wie sie die Website behandeln sollen.
+
+Kopiere im Rahmen dieser Anleitung den folgenden Code-Schnipsel in die Datei:
+
+```
+# Beispiel: Erlaube allen Suchmaschinen, diese Website zu scannen
+# und in den Suchindex aufzunehmen.
+# Volle Syntax: https://developers.google.com/search/docs/advanced/robots/create-robots-txt
+User-agent: *
+Allow: /
+```
+
+
+## 5. Erstelle `astro.config.mjs`
+
+Astro wird über die Datei `astro.config.mjs` konfiguriert. Diese Datei ist optional, aber für den Fall, dass du später Änderungen an der Konfiguration vornehmen willst, kannst du sie schon jetzt erstellen.
+
+Erstelle die Datei `astro.config.mjs` im Stammverzeichnis deines Projekts und kopiere den folgenden Code hinein:
+
+```
+import { defineConfig } from 'astro/config';
+
+// https://astro.build/config
```

Comment:
Maybe change the link to "https://docs.astro.build/de/reference/configuration-reference/" so it uses the german variant instead of the english one

Reply:
Gute Idee! Wird übernommen :)